### PR TITLE
Add punet shapes and vmfb hash to csv data.

### PR DESCRIPTION
**Pull request by saienduri**
This commit adds punet transpose b shapes to the gemmbench problem set and vmfb hash for every kernel to csv data, so we can eliminate noise and only look at different vmfbs.

### CODE REVIEW

**Review by kuhar — CHANGES_REQUESTED** [review]

File: common_tools/utils/bench_utils.py [hunk 1]
```diff
@@ -9,6 +9,24 @@
...
+        for chunk in iter(lambda: file.read(4096), b""):
+            md5_hash.update(chunk)
+
+    # Get the digest (raw bytes) of the MD5 hash
+    digest = md5_hash.digest()
+
+    # Encode the digest in Base64 and decode to string
+    base64_hash = base64.b64encode(digest).decode('utf-8')
+
+    return base64_hash
```
[the remark this answers is not in the document]
Comment (saienduri) [reply common_tools/utils/bench_utils.py:29]: This is the format that Azure uses for storage (base 64 encoded md5 hash):
image
I wanted to keep consistent with that. Maybe we can change in the libtuner to do the same as here

File: gemmbench/problems.py [hunk 2]
```diff
@@ -635,13 +635,23 @@ def is_compute_bound(M, N, K, bpe):
...
     (8192, 5120, 640),
+    # PUNET
+    (64, 1280, 2048),
+    (64, 640, 2048),
+    (1024, 10240, 1280),
+    (1024, 1280, 1280),
+    (1024, 1280, 5120),
+    (4096, 640, 640),
+    (4096, 5120, 640),
+    (4096, 640, 2560)
```
[the remark this answers is not in the document]
Comment (saienduri) [reply gemmbench/problems.py]: Sure, sorted now

File: common_tools/utils/bench_utils.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (kuhar) [reply common_tools/utils/bench_utils.py:29]: I think it's better to use the same format that `md5sum` uses in the CLI -- I doubt we will be comparing hashes with anything in Azure

[the remark this answers is not in the document]
Comment (kuhar) [reply common_tools/utils/bench_utils.py:29]: base64 also makes the hash longer

[the remark this answers is not in the document]
Comment (saienduri) [reply common_tools/utils/bench_utils.py]: Ok using same as libtuner

**Review by kuhar — APPROVED** [review]

**Pull request merged**